src/index.js

```diff
@@ -50,18 +53,21 @@ class DropdownTreeSelect extends Component {
     inlineSearchInput: PropTypes.bool,
     tabIndex: PropTypes.number,
     disablePoppingOnBackspace: PropTypes.bool,
+    disableKeyboardNavigation: PropTypes.bool,
```

Comment:
What's the use case for this? At a glance it seems like an a11y blocker but people don't have to use the keyboard if they don't want to so curios as to why this is needed.

Reply:
From my PR description above:

> The new prop disableKeyboardNavigation allows disabling all actions associated with a key down event in the search input box. This is important if you need an input box to behave normally. For instance, when I press SHIFT+HOME, it should select the entire line, so I can press BACKSPACE to clear it.

I can't use arrow keys either on the search box right now.

Reply:
> an input box to behave normally

I hear you but it's not really a regular input box - hence the special treatment. 

> when I press SHIFT+HOME, it should select the entire line, so I can press BACKSPACE to clear it.

Can this handled by adding additional functionality to the keyboard navigation logic? So that we don't have to choose between having it or not at all?